digi/xbee/recovery.py

```diff
@@ -80,6 +80,19 @@
     0xfc: [921600],  # When receiving data (explicit)
 }
 
+_RECOVERY_ALL_REASONABLE_BAUDRATES = [ 921600,
```

Comment:
To make recovery of a device that at typical defaults a bit faster, I might reorder these so 9600 is the first entry here.